wire-schema/src/main/java/com/squareup/wire/schema/MarkSet.java

```diff
@@ -83,9 +83,17 @@ boolean mark(ProtoType type) {
   boolean mark(ProtoMember protoMember) {
     if (protoMember == null) throw new NullPointerException("type == null");
     if (identifierSet.excludes(protoMember)) return false;
-    return members.containsKey(protoMember.type())
+    boolean result = members.containsKey(protoMember.type())
```

Comment:
this should probably be something other than `result` because it isn’t. Maybe `newMark` ?